torch/_dynamo/compiled_autograd.py

```diff
@@ -147,7 +147,10 @@ def post_acc_grad_hook(self, inputs, hook_id):
             inputs,
         )
         with disable_proxy_modes_tracing():
-            inputs = [maybe_clone(x) for x in inputs]
+            if isinstance(inputs, list):
```

Comment:
This seems weird. Naively, I would expect an invariant that this is only ever called with a list of inputs. When is it not called with list of inputs? Also cc @jansel

Reply:
This is actually never called with a list of inputs. I can remove that.